logstash-core/spec/logstash/outputs/base_spec.rb

```diff
@@ -15,6 +15,14 @@ def receive(event)
   end
 end
 
+class LogStash::Outputs::NOOPLegacyNoWorkers < ::LogStash::Outputs::Base
+  LEGACY_WORKERS_NOT_SUPPORTED_REASON = "legacy reason"
+
+  def register
+    workers_not_supported(LEGACY_WORKERS_NOT_SUPPORTED_REASON)
+  end
+end
+
```

Comment:
I think we are missing a test for the `declare_workers_not_supported!` We have a test for the legacy method but not the new one?